pkgs/development/python-modules/outlines/default.nix

```diff
@@ -48,16 +52,19 @@ buildPythonPackage rec {
     scipy
     torch
     transformers
+    pycountry
+    pyairports
   ];
 
-  pythonImportsCheck = [ "outlines" ];
+  checkPhase = ''
+    export HOME=$(mktemp -d)
+    python3 -c 'import outlines'
```

Comment:
This is `pythonImportsCheck` with `preCheck = ''export HOME=..''`

Reply:
I tried that first and it did not work, I don't think `preCheck` applies to the import check.

Reply:
preInstallCheck maybe

Reply:
I think given the rest is good to go that we can leave this for now as it works, it can be improved in a future PR.